server/routes/temporary-accommodation/manage.ts

```diff
@@ -308,5 +308,9 @@ export default function routes(controllers: Controllers, services: Services, rou
     auditEvent: 'CONFIRM_ASSESMENT_STATUS_CHANGE',
   })
 
+  put(paths.assessments.update.pattern, assessmentsController.update(), {
+    auditEvent: 'UPDATE_ASSESSMENT',
+  })
```

Comment:
My feeling is use the same conventions as other `puts`, and use `redirectAuditEventSpecs`, even though we only have the one possible redirect destination